js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,29 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="targetContextAttributes"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="targetContextAttributes"
+          />
+        }
+      >
+        <Controller
+          name="targetContextAttributes"
+          defaultValue={false}
+          control={control}
+          render={({ field }) => (
+            <Checkbox
+              id="targetContextAttributesCheckbox"
```

Comment:
The `id` prop passed to `Checkbox` needs to match the one provided as `fieldId` to the `FormGroup`, as otherwise the input is not associated with the label.

This creates problems for visually impaired users, as their accessibility tools are not able to understand the structure. It also handles other behavior, such as focusing the input when the label is clicked.

```suggestion
              id="targetContextAttributes"
```